sdk/core/src/services/WalletStorageService.ts

```diff
@@ -524,6 +530,62 @@ export default class WalletStorageService {
       limit: limit || 100,
     }
   }
+
+  async saveUnencryptedCredentials(data: unknown[], storageRegion?: string) {
```

Comment:
@inga-affinidi still see this name convention. Lets have it as encryptAndSaveCredentials (or something like that, to avoid confusion <its gives assumption that credentials will be stored at unencrypted way>)